var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -14,6 +14,7 @@ class PyBglibpy(PythonPackage):
 
     version('develop', branch='master')
 
+    version('4.3.19', commit='bce00a1ddedb605a5ed5225989192eb5f7e133ae')
```

Comment:
for module purpose we also have entry here: https://github.com/BlueBrain/spack/blob/2c8bdeeff148c28fb280adfd9bf7ecc518153012/deploy/packages/bbp-packages.yaml#L126

could you also update that one?

(By the way - CI will run after bb5 maintenance)

Reply:
Ok, done.

This process urgently needs to be automated. I'll definitely work on BGLibPy automating a PR when a new version is released.
Is there maybe any plan to e.g. let spack automatically add new python packages when they're released to devpi?

Reply:
(Btw, about this bbp-packages file? Is there way to specify that it should automatically pick the latest version?)

Reply:
@wvangeit with the current setup, dropping the version number from `bbp-packages.yaml` will keep modules for all versions of `py-bglibpy` in the deployment archives.

I have to test it still, but in the next deployment iteration, we should be able to drop the version specification and just list software to be installed.